File: ReservationServiceClient/model/reservationservice/base/Review.cs

public class Review {
    public long id{ get; set; }
    public long client_user_id{ get; set; }
    public long firm_id{ get; set; }
    public string text{ get; set; }
    public double grade{ get; set; }

    public Review() {
    }

    public Review(long id, long client_user_id, long firm_id, string text, double grade) {
        this.id = id;
        this.client_user_id = client_user_id;
        this.firm_id = firm_id;
        this.text = text;
        this.grade = grade;
    }
}

File: reservation-service/src/main/java/rs/raf/reservationservice/domain/Review.java
package rs.raf.reservationservice.domain;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "review")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Review {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "client_user_id", nullable = true)
    private Long client_user_id;
    @ManyToOne
    private Firm firm;
    @Column(name = "text")
    private String text;
    @Column(name = "grade")
    private Double grade;
}
